pkg/image/fetcher.go

```diff
@@ -121,10 +121,9 @@ func (f *Fetcher) Fetch(ctx context.Context, name string, options FetchOptions)
 		// FIXME: this matching is brittle and the fallback should be removed when https://github.com/buildpacks/pack/issues/2079
 		// has been fixed for a sufficient amount of time.
 		// Sample error from docker engine:
-		// `image with reference <image> was found but does not match the specified platform: wanted linux/amd64, actual: linux`
-		if strings.Contains(err.Error(), "does not match the specified platform") &&
-			(strings.HasSuffix(strings.TrimSpace(err.Error()), "actual: linux") ||
-				strings.HasSuffix(strings.TrimSpace(err.Error()), "actual: windows")) {
+		// `image with reference <image> was found but does not match the specified platform: wanted linux/amd64, actual: linux` or
+		// `image with reference <image> was found but its platform (linux) does not match the specified platform (linux/amd64)`
+		if strings.Contains(err.Error(), "does not match the specified platform") {
```

Comment:
Is this meant as a quick fix? Because it would break again if docker decides to change the wording. But a quick fix would be highly appreciated since this is really critical for us.